dev/seed.py

```diff
@@ -17,7 +19,39 @@ def get_db_path():
     return os.path.join(data[0], "scheduler.sqlite")
 
 
-def create_random_job(index: int):
+def create_random_job_def(index: int):
+    name = random.choice(
+        [
+            "definition alpha",
+            "definition beta",
+            "super schedule",
+            "cool definition",
+            "lame definition",
+        ]
+    )
+    job_definition_id = str(uuid4())
+    input_uri = "".join(name.split()) + ".ipynb"
+    output_prefix = ""
+    timezone = random.choice(list(zoneinfo.available_timezones()))
+    # random schedules can be generated via https://crontab.guru/
+    schedule = random.choice(
+        ["0 0,12 1 */2 *", "0 4 8-14 * *", "0 0 1,15 * 3", "@weekly", "5 0 * 8 *", "15 14 1 * *"]
```

Comment:
Is "@weekly" a failure case?

Reply:
It was generated by crontab.guru, but it looks like it's non-standard. I'll remove it from the schedule stubs.